### DISCUSSION TITLE
How to set t_eval ?

### DISCUSSION THREAD

**Discussion by Mrzhang-hub** [pybamm-team/PyBaMM#1963] [Q&A]
I wanna ask how to set t_eval when only one current value from driving cycle is taken as input each time. It seems it is only a specific time point instead of an time interval.   Is it reasonable that I randomly set a very small value for t_eval such as 0.01s?  But when I do this , the output value of SEI thickness will fluctuate instead of increasing as battery ages

![微信图片_20220303212234](https://user-images.githubusercontent.com/59716473/156573479-c3e7ae4c-b42c-4b5d-a36d-4da3876a046d.png)

**Comment by rtimms** [comment]
You can just do e.g.
```python3
t_eval = np.linpace(0, 100, 100)
sim.solve(t_eval)
```

If you are using a drive cycle and you don't pass `t_eval` it will just used the times in the csv file, but you can pass your own `t_eval` (you might get a warning if your `t_eval` doesn't contain all the times in the data or if it is at a lower sample rate).

**Reply by rtimms** [reply]
Also, what is the scale on that plot? I can't quite make it out, but it looks like the fluctuations are on the order of 10^-17 around a base value of 5e-9, so I think this is just numerical.
